support-frontend/assets/pages/[countryGroupId]/student/helpers/studentFAQs.tsx

```diff
@@ -76,25 +78,25 @@ const auSupporterPlusFAQ: (regionName: string) => FAQItem[] = () => [
 		),
 	},
 	{
-		title: 'What is included in my All-access subscription?',
+		title: `What is included in my ${supporterPlusLabel} subscription?`,
 		body: (
 			<p>
-				Your All-access digital subscription entitles you to all the benefits
+				Your {supporterPlusLabel} subscription entitles you to all the benefits
 				listed above, including: unlimited access to the Guardian news app and
 				Guardian Feast app, ad-free reading on all your devices, exclusive
 				newsletter for supporters and far fewer asks for support. Currently,
-				this offer provides you with a free All-access digital subscription for
-				a period of 24 months after redemption. We will be in touch during this
-				period if there are opportunities to extend beyond this timeframe.
+				this offer provides you with a free {supporterPlusLabel} subscription
+				for a period of 24 months after redemption. We will be in touch during
+				this period if there are opportunities to extend beyond this timeframe.
 			</p>
 		),
 	},
 	{
-		title: 'Will my All-access subscription work across all devices?',
+		title: `Will my ${supporterPlusLabel} subscription work across all devices?`,
```

Comment:
I'm sure it's fine, but I guess this means a small copy change? Was `All-access` and will now be `All-access digital`.